android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/AdjustKeyboardHeightActivity.java

```diff
@@ -77,9 +73,7 @@ protected void onCreate(Bundle savedInstanceState) {
     resetButton.setOnClickListener(new View.OnClickListener() {
       public void onClick(View v) {
         // Clear the keyboard height preferences to reset
-        editor.remove(KMManager.KMKey_KeyboardHeightPortrait);
-        editor.remove(KMManager.KMKey_KeyboardHeightLandscape);
-        editor.commit();
+        KMManager.applyKeyboardHeight(context, 0);
```

Comment:
Instead of using `0` here, can we add `KMManager.DEFAULT_KEYBOARD_HEIGHT`?

Reply:
Instead of "default height" in the constant name, I'd prefer defining something like 
RESET_KEYBOARD_HEIGHT = 0;

Reply:
I'm not sure I completed this as intended. Please check this.

Reply:
Ok, I'm sure I didn't complete this as intended. KMManager doesn't have any variable called with all caps, so I don't have an example. Please fix this on both the activity and KMManager.